api/src/test/java/org/openmrs/validator/AllergyValidatorTest.java

```diff
@@ -177,5 +178,20 @@ public void validate_shouldPassForAValidAllergy() {
 		validator.validate(allergy, errors);
 		
 		assertFalse(errors.hasErrors());
+	}
+    @Test
```

Comment:
Add this on the same line, correct the identation as well.

Reply:
Like how ....I think I didnt get you right @sherrif10

Reply:
```suggestion
    @Test
    public void valid_shouldRejectNumericReactionValue() throws Exception { 
    
    }
```

Reply:
It should be on same the line as above